modules/clipboard.js

```diff
@@ -546,6 +550,12 @@ function matchTable(node, delta) {
   return applyFormat(delta, 'table', row);
 }
 
+function matchPlainText(node, delta) {
+  let text = node.data || '';
```

Comment:
```suggestion
  let text = node.data ?? '';
```

Reply:
It isn't actual in this case. Left-hand value cannot be `false` or `0`